Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -68,6 +71,28 @@
 
 #define BOARD_NAME         "Printrboard Rev F"
 #define LARGE_FLASH        true
+#ifndef USBCON
+  #error "USBCON should be defined by the platform for this board."
+#endif
+
```

Comment:
And just tested BRAINWAVE (AT90USB647) with @Bob-the-Kuhn 's Marlin extension and it too compiled as expected w/o USBCON defined in the board pins file (of course the resulting sketch wouldn't fit, but i'm sure it could be made to).

Submitted #7347 for this to save you a few seconds in case this way you want to go Scott.  If this isn't correct, just reject.